### PR TITLE
Let users be sudoers

### PR DESCRIPTION
**Pull request by paolo-crisafulli**
To let  users use additional tools, the way to go is to add lines in extras.dockerfile.
Nevertheless, the process of discovering the tools you need, then terminate the user container, add lines to extras.dockerfile, build a new image and run a new container is very tedious - especially doing that one tool at a time, which is what happens in practice.
So I suggest to let users be sudoers, in order to let them install new tools while using a container, then report them to extras.dockerfile for later use.
To sum up, it's a suggested usability improvement.

### CODE REVIEW

**Comment by LukeMondy** [comment]
Cool, thanks again!
I merged your commit internally, and added some extra stuff around it to make it clear that using sudo is ephemeral. It should be pushed out later today.
